doc/supportedPlatformVerions.md

```diff
@@ -170,6 +170,10 @@
 - 15.2.0
 - 15.2.1
 - 15.3.0
+- 16.5.0
```

Comment:
do we support all these? I see only 16.13 now

Reply:
Removed. Those were added by our script which automatically populates versions